common/persistence/class.SqlKvDriver.php

```diff
@@ -74,54 +77,53 @@ public function set($id, $value, $ttl = null, $nx = false)
             throw new common_exception_NotImplemented('NX not implemented in '.__CLASS__);
         }
         try{
-            
+
             $expire = is_null($ttl) ? 0 : time() + $ttl;
-            
+
             $encoded = base64_encode($value);
             $platformName = $this->sqlPeristence->getPlatForm()->getName();
             $params = array(':data' => $encoded, ':time' => $expire, ':id' => $id);
-            
-            
+
+
             if($platformName == 'mysql'){
                 //query found in Symfony PdoSessionHandler
                 $statement = "INSERT INTO kv_store (kv_id, kv_value, kv_time) VALUES (:id, :data, :time) 
                     ON DUPLICATE KEY UPDATE kv_value = VALUES(kv_value), kv_time = VALUES(kv_time)";
                 $returnValue = $this->sqlPeristence->exec($statement,$params);
-                
+
             } else if($platformName == 'oracle'){
                 $statement = "MERGE INTO kv_store USING DUAL ON(kv_id = :id) 
```

Comment:
Strange thing: for oracle, statement isn't executed. Is it ever used?

Reply:
I have doubts if oracle compatibility was tested recently, at all.